modules/gitops-using-argo-cd-instance-to-manage-cluster-scoped-resources.adoc

```diff
@@ -6,7 +6,7 @@
 [id="using-argo-cd-instance-to-manage-cluster-scoped-resources{context}"]
 = Using an Argo CD instance to manage cluster-scoped resources
 
-To manage cluster-scoped resources, update the existing `Subscription` object for the `gitops-title` Operator and add the namespace of the Argo CD instance to the `ARGOCD_CLUSTER_CONFIG_NAMESPACES` environment variable in the `spec` section.
+To manage cluster-scoped resources, update the existing `Subscription` object for the {gitops-title} Operator and add the namespace of the Argo CD instance to the `ARGOCD_CLUSTER_CONFIG_NAMESPACES` environment variable in the `spec` section.
 
 [discrete]
```

Comment:
This change looks good to me. 
I just noticed and wondered why the [discrete] heading style is used for this procedure? I don't think it has any function here. But otherwise lgtm :+1: